jetty-http2/http2-http-client-transport/src/main/java/org/eclipse/jetty/http2/client/http/HttpReceiverOverHTTP2.java

```diff
@@ -285,7 +284,7 @@ private void process(boolean resume)
                 return;
 
             // Process only if there is demand.
-            synchronized (this)
+            try (AutoLock ignored = lock.lock())
```

Comment:
why `ignored` ?

Reply:
Because otherwise IDEs report it as a warning because it's an unused variable.

Reply:
As I said below (above), that's a problem with the IDE.  The variable is used in the virtual finally block.  Let's just call it lock and raise a bug on the ide